portal/src/main/webapp/js/src/cc-plots/cross-cancer-plotly-plots.js

```diff
@@ -487,80 +487,78 @@ var ccPlots = (function (Plotly, _, $) {
+                    final_pdf_str = final_pdf_str.replace(/text-anchor='middle'/g, "");
+                    final_pdf_str = final_pdf_str.replace(/text-anchor: start;/g, "");
+                    final_pdf_str = final_pdf_str.replace(/font-family: 'Open Sans',/g, "");
+                    final_pdf_str = final_pdf_str.replace(/fill: transparent;/g, "fill-opacity: 0;");
+                    final_pdf_str = final_pdf_str.replace(/'/g, "\"");
+
+                    var downloadOptions = {
+                        filename: "cross-cancer-plots.pdf",
+                        contentType: "application/pdf",
+                        servletName: "svgtopdf.do"
+                    };
+                    cbio.download.initDownload(final_pdf_str, downloadOptions);
+
+                });
+                $("#cc_plots_data_download").click(function() {
+                    var get_tab_delimited_data = function() {
+                        var result_str = "Sample Id" + "\t" + "Cancer Study" + "\t" + "Profile Name" + "\t" + "Gene" + "\t" + "Mutation" + "\t" + "Value" + "\n";
+                        _.each(formatted_data, function(_obj) {
+                            if ( _obj.sequenced) {
+                                if (_obj.mutation_type === "non" ) {
+                                    result_str += _obj.sample_id + "\t" + _obj.study_name + "\t" + "RNA Seq V2" + "\t" + gene[0] + "\t" + "Not Mutated" + "\t" + _obj.profile_data + "\n";
                                 } else {
-                                    result_str += _obj.sample_id + "\t" + _obj.study_name + "\t" + "RNA Seq V2" + "\t" + gene[0] + "\t" + "Not Sequenced" + "\t" + _obj.profile_data + "\n";
+                                    result_str += _obj.sample_id + "\t" + _obj.study_name + "\t" + "RNA Seq V2" + "\t" + gene[0] + "\t" + _obj.mutation_details + "\t" + _obj.profile_data + "\n";
                                 }
-                            });
-                            return result_str;
-                        };
-                        cbio.download.clientSideDownload([get_tab_delimited_data()], "plots-data.txt");
-                    });
-                    
-                    // fetch data and init view
-                    fetch_profile_data(_.pluck(_.pluck(window.studies.models, "attributes"), "studyId"));
-                }
-            }
+                            } else {
+                                result_str += _obj.sample_id + "\t" + _obj.study_name + "\t" + "RNA Seq V2" + "\t" + gene[0] + "\t" + "Not Sequenced" + "\t" + _obj.profile_data + "\n";
+                            }
+                        });
+                        return result_str;
+                    };
+                    cbio.download.clientSideDownload([get_tab_delimited_data()], "plots-data.txt");
+                });
+
+                // fetch data and init view
+                fetch_profile_data(Object.keys(_studySampleMap));
```

Comment:
Just give Karthik a heads up, the only change in this block is `fetch_profile_data(Object.keys(_studySampleMap));` from `fetch_profile_data(_.pluck(_.pluck(window.studies.models, "attributes"), "studyId"));`